src/components/protocol_handler/test/protocol_handler_tm_test.cc

```diff
@@ -192,17 +202,27 @@ class ProtocolHandlerImplTest : public ::testing::Test {
     // use protection OFF
     const bool callback_protection_flag = PROTECTION_OFF;
 #endif  // ENABLE_SECURITY
+    std::vector<std::string> empty;
```

Comment:
OK, since 'empty' was a local variable, it could be deleted before NotifySessionStartedResult() was called.
I've changed it to a member variable in https://github.com/smartdevicelink/sdl_core/pull/1700/commits/0f937b880bd48c1838cdfbb306d4c39057a89eb2. Could you kindly check if you still see the issue?
Thanks.